app/main.ts

```diff
@@ -1,7 +1,15 @@
 import { bootstrap }    from '@angular/platform-browser-dynamic';
-
+import { HTTP_PROVIDERS } from "@angular/http";
 import { AppComponent } from './app.component';
-
 import { appRouterProviders } from "./app.routes";
 
-bootstrap(AppComponent,[appRouterProviders]);
+
```

Comment:
Sorry, you have been talking about the http module, this module provide any type of http request , i think it might be useful simulate a web api, there is the api reference for this module https://angular.io/docs/ts/latest/api/http/index/Http-class.html